physics/physpkg.F90

```diff
@@ -1381,7 +1410,11 @@ subroutine tphysac (ztodt,   cam_in,  &
     use lunar_tides,        only: lunar_tides_tend
     use ssatcontrail,       only: ssatcontrail_d0
     use microp_driver,      only: microp_driver_tend
+#ifdef OSLO_AERO
+    use oslo_aero_microp,   only: oslo_aero_microp_run
```

Comment:
Because its totally different. And I wanted to have all the oslo_aero code to have a prefix of oslo_aero to eventually think of putting this in another model rather than having a name which matches cam.